docs/modules/ROOT/pages/tutorials/installation-openshift.adoc

```diff
@@ -0,0 +1,99 @@
+# Install vcluster on OpenShift
+
+This page guides through the installation process of vcluster on OpenShift and sets up authentication with OIDC using Keycloak.
+
+At the end of this tutorial we will have:
+
+* a vcluster instance running on OpenShift
+* vcluster's Kubernetes API exposed with an Ingress
+* Let's Encrypt certificates provisioned by `cert-manager`
+* OIDC authentication
+
+== Prerequisites
+
+* Admin access to existing Keycloak instances
+* oidc-plugin for kubectl (https://github.com/int128/kubelogin)
+* `kubectl`
+* `base64`
+* `jq`
+
+== Deploy vcluster and Ingress
+
+. Configure catalog
++
+[source,yaml]
+----
+parameters:
+  vcluster:
+    host_service_cidr: 172.30.0.0/16
```

Comment:
I'd add a reference to https://hub.syn.tools/vcluster/references/parameters.html#_host_service_cidr here. Since it's quite non-obvious what to do.